cmake/haero.cmake.in

```diff
@@ -45,8 +40,13 @@ add_library(kokkos INTERFACE)
 target_link_libraries(kokkos INTERFACE kokkoscontainers kokkoscore)
 
 if (HAERO_ENABLE_GPU)
-  # FIXME: this assumes an NVIDIA GPU!
-  set(HAERO_LIBRARIES @HAERO_LIBRARIES@;kokkos;cuda;m)
+  if (HAERO_AMD_GPU)
+    set(HAERO_LIBRARIES @HAERO_LIBRARIES@;kokkos;rocm;m)
+  else()
+    set(HAERO_CXX_COMPILER @CMAKE_INSTALL_PREFIX@/bin/nvcc_wrapper)
```

Comment:
Doesn't HAERO link against kokkos? As such, you should not need to set the compiler to nvcc wrapper. Everything linking against kokkos gets kokkos_launch_compiler injected int the build rule...